python/ray/util/tracing/enable_tracing.py

```diff
@@ -0,0 +1,18 @@
+import os
+from opentelemetry import trace
+from opentelemetry.sdk.trace import TracerProvider
+from opentelemetry.sdk.trace.export import (
+    ConsoleSpanExporter,
+    SimpleExportSpanProcessor,
+)
+from typing import Any
+
+
+def _setup_tracing(*args: Any, **kwargs: Any) -> None:
```

Comment:
What is this file for? Is it a default that users can use without providing a hook or something? Could you leave a header comment in the file?